app/src/app/components/header/HeaderDropDown.tsx

```diff
@@ -2,19 +2,36 @@ import React from "react";
 import {NavDropdown} from "react-bootstrap";
 import AccountCircleIcon from "@mui/icons-material/AccountCircle";
 import LogoutIcon from "@mui/icons-material/Logout";
+import {useSelector} from "react-redux";
+import {RootState, useAppDispatch} from "../../../types";
+import {logout} from "../../store/login/login";
+import {useNavigate} from "react-router-dom";
 
 export default function HeaderDropDown() {
+    const navigate = useNavigate();
+    const dispatch = useAppDispatch();
+    const {isAuthenticated} = useSelector((state: RootState) => state.login);
+
+    const handleLogout = () => {
+        dispatch(logout());
+        navigate("/login");
+    };
+
     return (
-        <NavDropdown data-testid="drop-down" className="icon-primary" title={
-            <AccountCircleIcon sx={{fontSize: 35}}/>}>
-            <NavDropdown.Item>l.ani@imperial.ac.uk</NavDropdown.Item>
-            <NavDropdown.Item>Manage access</NavDropdown.Item>
-            <NavDropdown.Item>Publish packets</NavDropdown.Item>
-            <NavDropdown.Divider></NavDropdown.Divider>
-            <NavDropdown.Item>
-                <LogoutIcon className="me-2" sx={{fontSize: 20}}/>
-                Logout
-            </NavDropdown.Item>
-        </NavDropdown>
+        <>
+            {isAuthenticated &&
+                <NavDropdown data-testid="drop-down" className="icon-primary" title={
+                    <AccountCircleIcon sx={{fontSize: 35}}/>}>
+                    <NavDropdown.Item>l.ani@imperial.ac.uk</NavDropdown.Item>
```

Comment:
Could replace this with the real user name or email now I guess?

Reply:
Oh yes, I will vote to do that in a different ticket. https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4638/Validate-and-extract-user-details-from-Token